can/map/test-real-time-super-model.js

```diff
@@ -11,6 +11,8 @@ var map = [].map;
 var later = testHelpers.later;
 var queues = require("can-queues");
 var makeRealTimeSuperModel = require("./make-real-time-super-model");
+var constructorStore = require("../../constructor/store/store");
+constructorStore.requestCleanupDelay = 1;
```

Comment:
why was this needed? it's also part of  `can/map/map_test.js` and needed there also